web/src/main/angular/src/app/core/components/transaction-id-search/transaction-id-search-container.component.css

```diff
@@ -0,0 +1,35 @@
+:host {
+    display: block;
+    width: 170px;
+    height: 32px;
+    position: relative;
+}
+.l-search-group-wrap {
+    width: 100%;
+    height: 100%;
+}
+
+.l-search-input {
+    width: 100%;
+    height: 100%;
+    border: 1px solid #d7dde4;
+    padding: 0 5px 0 5px;
```

Comment:
Adjusting padding will be better for the consistency with other search-input
`padding: 0 28px 0 10px`

Plus, please add text-overflow ellipsis:
```
text-overflow: ellipsis;
white-space: nowrap;
overflow: hidden;
```